packages/kit/src/ignore.ts

```diff
@@ -32,3 +34,29 @@ export function isIgnored (pathname: string): boolean {
   }
   return !!(relativePath && nuxt._ignore.ignores(relativePath))
 }
+
+// Internal
+
+/**
+ * This function turns string containing groups '**\/*.{spec,test}.{js,ts}' into an array of strings.
+ * For example will '**\/*.{spec,test}.{js,ts}' be resolved to:
+ * ['**\/*.spec.js', '**\/*.spec.ts', '**\/*.test.js', '**\/*.test.ts']
+ *
+ * @param group string containing the group syntax
+ * @returns {string[]} array of strings without the group syntax
+ */
+export function resolveGroupSyntax (group: string): string[] {
```

Comment:
We should update [index.ts](https://github.com/nuxt/nuxt/blob/ae9a98b942fc43f92ed939c3b96d275f3be43907/packages/kit/src/index.ts#L15) to use named exports from this file (excpluding internal util) to avoid exposing internals to final package.

Reply:
Agreed 👍🏻